src/FixedDataTableCellGroup.js

```diff
@@ -73,21 +131,40 @@ class FixedDataTableCellGroupImpl extends React.Component {
     this._initialRender = false;
   }
 
+  /**
+   * Returns Object consisting of keys and widths of the columns in the current cell group.
+   * @returns {{keys: [], widths: []}}
+   */
+  getCellGroupWidth = () => {
+    const { columns } = this.props;
+    const cellGroupColumnWidths = {
+      keys: [],
+      widths: []
+    };
+    if (this.props.isHeader) {
+      for (let i = 0, j = columns.length; i < j; i++) {
+        const key = columns[i].props.columnKey || 'cell_' + i;
+        cellGroupColumnWidths.keys.push(key);
+        cellGroupColumnWidths.widths.push(columns[i].props.width);
+      }
+    }
+    return cellGroupColumnWidths;
+  }
+
+
   render() /*object*/ {
     var props = this.props;
     var columns = props.columns;
     var cells = new Array(columns.length);
     var contentWidth = sumPropWidths(columns);
 
-    var isColumnReordering = props.isColumnReordering && columns.reduce(function (acc, column) {
-      return acc || props.columnReorderingData.columnKey === column.props.columnKey;
-    }, false);
-
     var currentPosition = 0;
     for (var i = 0, j = columns.length; i < j; i++) {
       var columnProps = columns[i].props;
       var cellTemplate = columns[i].template;
-      var recyclable = columnProps.allowCellsRecycling && !isColumnReordering;
+     /* Todo(deshpsuy): recyclable should always be false while reordering. But since now reordering is decoupled out of FDT.
```

Comment:
@pradeepnschrodinger do you have any idea why this was true?  Was it just to avoid rerendering a cell we were going to drop wherever we dragged too, and thus this could save a bit of perf?  Or are there larger bugs which would happen if it was recycled?

Reply:
Sounds like an issue that needs to be addressed.  

Could you elaborate on "reordering won't work as expected"?  What happens?  Do we throw an error?  Does some unexpected behavior occur?  And do we know why?

Reply:
As soon as the column moves out of view, it gets destroyed and the cell (which we are reordering) destroys as well resulting in sudden end of reordering.

Reply:
Is this because the native drag drop stops when the element being dragged is removed from the DOM?  I wonder if we could do something to work around that.

Reply:
On additional looks, it seems like this is probably due to the tracking of state for the reorder happening in the ResizeReorderCell.  Do you think we could solve this by storing that state outside of the Cell?